requirements/surfacemesh.txt

```diff
@@ -0,0 +1,6 @@
+# surfacemesh
+
+trimesh
+networkx
+rtree
+
```

Comment:
yeah, trimesh subsumes stl

Reply:
Instead of installing rtree, I opted out for using the shapely in-built rtree in our [mesher](https://github.com/flexcompute/tidy3d/blob/ba0f62a227192e2e8ff851c38d3e1ae4412dd58a/tidy3d/components/grid/mesher.py#L13). Maybe that can work for you too?

Reply:
Actually rtree doesn't seem to be used in the code - is it an optional requirement of one of the other two requirements?